src/core/stats.cpp

```diff
@@ -69,10 +193,22 @@ void Stats::dump(bool includeZeros) {
 
     std::sort(pairs.begin(), pairs.end());
 
+    uint64_t accumulated_stat_timer_ticks = 0;
     for (int i = 0; i < pairs.size(); i++) {
-        if (includeZeros || (*counts)[pairs[i].second] > 0)
-            fprintf(stderr, "%s: %ld\n", pairs[i].first.c_str(), (*counts)[pairs[i].second]);
+        if (includeZeros || (*counts)[pairs[i].second] > 0) {
+            if (startswith(pairs[i].first, "us_")) {
+                fprintf(stderr, "%s: %lu\n", pairs[i].first.c_str(),
+                        (uint64_t)((*counts)[pairs[i].second] / cycles_per_us));
+
+            } else
+                fprintf(stderr, "%s: %lu\n", pairs[i].first.c_str(), (*counts)[pairs[i].second]);
+
+            if (startswith(pairs[i].first, "us_timer"))
+                accumulated_stat_timer_ticks += (*counts)[pairs[i].second];
+        }
```

Comment:
Could we do something ilke `if (pairs[1].first == "ticks_in_main") ticks_in_main = pairs[i].second;` and then later just check accumulated_stat_timer_ticks against it?